core/src/main/java/io/github/belgif/rest/guide/validator/core/OpenApiViolationAggregator.java

```diff
@@ -125,4 +128,20 @@ private boolean fileShouldBeExcluded(Path relativePath) {
         return false;
     }
 
+    public static OpenApiViolationAggregator aggregate(List<OpenApiViolationAggregator> aggregators) {
+        Set<Violation> aggregatedViolations = new HashSet<>();
+        List<String> aggregatedExcludedFiles = new ArrayList<>();
+        for (OpenApiViolationAggregator aggregator : aggregators) {
+            aggregatedViolations.addAll(aggregator.getViolations());
+            if(aggregatedExcludedFiles.isEmpty()) {
```

Comment:
no need to set excluded files on returned aggregate

Reply:
Even though the excluded files on this aggregated report aren't used anywhere now, I've added the aggregated excludedFiles in, just for clarity in possible future use. Also, currently, the excludedFiles are the same for each file that is processed, since there is only one configuration option.